IETF-RFC.md

```diff
@@ -139,14 +139,31 @@ related concepts from OAuth [RFC6749] and elsewhere:
   OCM API Discovery.
 * __Discoverable Server__ - A server that tries to supply information in
   OCM API Discovery.
-* __OCM Address__ - A string of the form
-  `<Receiving Party's identifier>@<fqdn>` which can be used to uniquely
-  identify a user or group "at" an OCM Server and MAY be referred to as
-  Federated Cloud ID.
-  `<Receiving Party's identifier>` is an opaque string, unique at the
-  server.  `<fqdn>` is the Fully Qualified Domain Name by which the
-  server is identified.  This MUST be the domain at which the
-  `/.well-known/ocm` endpoint of that server is hosted.
+* __OCM Address__ - identifies a user or group "at" an OCM Server.
+The OCM Address contains a server specific Party identifier, a host
+locating the OCM Server and an optional port. The OCM Address is not a
+URI as it does not have scheme and the identifier may contain reserved
+characters.
+
+        ocm-address = identifier "@" host [ ":" port]
+
+    The identifier is an opaque, case-sensitive utf-8 string. It is 
+    separated from the host by the last "@" in the OCM Address.
```

Comment:
The fact that it is the `last` @-sign is important, since it is common for example for an email-address to be the local part of the identifier, maybe we should say that explicitly, e.g.:

```
It is separated from the host by the last "@" in the OCM Address. It is possible, and common, to have multiple @-signs in a OCM-address, e.g. when an email address is the local part of the address:
  nomen.nescio@example.org@ocm.example.org
```